src/components/Hero/Hero.stories.tsx

```diff
@@ -229,6 +229,45 @@ export const Campaign: Story = {
           subHeaderText={otherSubHeaderTextLong}
         />
       </div>
+      <div>
+        <Heading
+          id="campaign-hero-long-text"
+          text="Campaign Hero with backdrop background color"
+        />
+        <Hero
+          backdropBackgroundColor="section.research.primary"
+          heroType="campaign"
+          heading={
+            <Heading
+              level="one"
+              id="campaign-hero-long-text-heading"
+              text="Hero Campaign"
+            />
+          }
+          imageProps={imageProps}
+          subHeaderText={otherSubHeaderTextLong}
+        />
+      </div>
+      <div>
+        <Heading
+          id="campaign-hero-long-text"
+          text="Campaign Hero with separate backdrop and foreground background color"
+        />
+        <Hero
+          backdropBackgroundColor="section.education.primary"
+          backgroundColor="#A03E31"
```

Comment:
Set this to a light color and then use the `foregroundColor` prop to set the text color to a dark color.